Modified fluxpulse_scope_sequence to allow for longer measurements

chellings opened the pull request:
A change by @christiankraglundandersen tested successfully during QAOA  experiments with @nathlacroix and @chellings and during measurements by @PAMouny 

Previously, the readout started only after the end of the flux pulse. Like this, the possible length of a measurement was T1-limited. By starting the readout already during the flux pulse, we can do flux pulse scope measurements with a pulse length of, e.g., 50us to get a better fit of the longest time constant in the IIR fitting. The change comes at the cost of a slight reduction of the contrast, but the above-mentioned experiments show that this does not impair the filter fitting procedure.

In addition, this pull request fixes the default value for an empty dict in fluxpulse_scope_sequence().

Inviting @stephlazar and @jo3141 to review.
FYI @antsr @MichKe95

nathlacroix committed:
modified fluxpulse_scope_sequence to allow for longer measurements

chellings committed:
fluxpulse_scope_sequence: avoid dict() in default values

stephlazar reviewed (approved):
Look good to me!
Maybe one small comment, you could consider making the ro_pulse delay an input parameter. Now it's hardcoded to 100ns, which should be fine, but ... well it's hardcoded :P

chellings committed:
fluxpulse_scope_sequence: made the pulse_delay or the readout pulse a parameter
to make the parameter accessible from outside, **kw was added to measure_flux_pulse_scope and is passed to the sequence function
